app/models/tipline_message.rb

```diff
@@ -0,0 +1,38 @@
+class TiplineMessage < ApplicationRecord
+  enum direction: { direction_unset: 0, incoming: 1, outgoing: 2 } # default: direction_unset
```

Comment:
Question for review: I'm wondering here if this abstraction makes sense, or if we should just directly store the trigger (which we still have saved in the model under payload)

Reply:
I think it makes sense because it helps a lot with code readability. We usually do that using constants to make the stored number to its meaning (for example: https://github.com/meedan/check-api/blob/develop/lib/check_archived_flags.rb). But it's OK to use an `enum` too.